server/verify.go

```diff
@@ -124,13 +124,14 @@ func VerifyAttestation(attestation *pb.Attestation, opts VerifyOpts) (*pb.Machin
 
 		// Parse event logs and replay the events against the provided PCRs
 		pcrs := quote.GetPcrs()
+		// This function also verifies TEE technology
```

Comment:
Yes and we should remove that. Add a TODO against an issue for now.

Reply:
removed